example/speech_recognition/LICENSE

```diff
@@ -0,0 +1,204 @@
+Copyright 2015-2016 Baidu USA LLC.  All rights reserved.
+Modifications copyright (C) 2017 Soonhwan Kwon, Kyusam Oh, Min Soo Kim, Jihyeon Seo, Junyoung Park, Seungkyung Kim, and Wanggeun Park
+
+                                 Apache License
+                           Version 2.0, January 2004
```

Comment:
@piiswrong would you tell me more details of implicity? I'm afraid that I'm confused about what you meant.